docs/current_docs/quickstart/853930-directories.mdx

```diff
@@ -118,6 +122,20 @@ You should see output similar to the following, which confirms that you've just
 dagger (registry.dagger.io/engine) linux/amd64
 ```
 
+Instead of exporting the entire directory to the Dagger host, you can export just the compiled binary:
+
+```shell
+dagger -m github.com/kpenfound/dagger-modules/golang@v0.1.5 call build --project=https://github.com/dagger/dagger --args=./cmd/dagger file --path=dagger export --path=./my-dagger-cli
```

Comment:
Should we explain the `file` command here as well since it hasn't been mentioned in the Getting Started guide before so could be confusing for a first time user?

Reply:
I see a lot of usage with `export --path=...`, when there's an alternative `--output=...`. While I don't like that there's two ways to do the same thing, the latter as some advantages:
- It's shorter to type, especially with the shorthand `-o`
- The output that it returns is more useful, i.e., absolute path of the saved resource on the host, instead of just `true`
- It respects the host's umask (hasn't been fixed in core `export` yet, see https://github.com/dagger/dagger/issues/6387)

Reply:
Modified